articles/energy-data-services/how-to-manage-data-security-and-encryption.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Microsoft Energy Data Services - Data Security and Encryption in Microsoft Energy Data Services #Required; page title is displayed in search results. Include the brand.
+description: Guide on security in Microsoft Energy Data Services and how to set up customer managed keys on Microsoft Energy Data Services #Required; article description that is displayed in search results. 
+author: sandeepchads #Required; your GitHub user alias, with correct capitalization.
+ms.author: sancha #Required; microsoft alias of author; optional team alias.
+ms.service: energy-data-services #Required; service per approved list. slug assigned by ACOM.
+ms.topic: how-to #Required; leave this attribute/value as-is.
+ms.date: 09/29/2022
+ms.custom: template-concept #Required; leave this attribute/value as-is.
+
+#Customer intent: As a developer, I want to set up customer-managed keys on Microsoft Energy Data Services
+---
+# Data Security and Encryption in Microsoft Energy Data Services
+
+This article provides an overview of security features in Microsoft Energy Data Services. It covers the major areas of [encryption at rest](../security/fundamentals/encryption-atrest.md), encryption in transit, TLS, https, microsoft-managed keys, and customer managed key. 
+
+## Encrypt data at rest
+
+Microsoft Energy Data Services uses several storage resources for storing metadata, user data, in-memory data etc. The platform uses service-side encryption to automatically encrypt all the data when it is persisted to the cloud. Data encryption at rest protects your data to help you to meet your organizational security and compliance commitments. All data in Microsoft Energy Data Services is encrypted with Microsoft-managed keys by default.
+In addition to Microsoft-managed key, you can use your own encryption key to protect the data in Microsoft Energy Data Services Preview. When you specify a customer-managed key, that key is used to protect and control access to the Microsoft-managed key that encrypts your data. 
+
+## Encrypt data in transit
+
+Microsoft Energy Data Services supports Transport Layer Security (TLS 1.2) protocol to protect data when it’s traveling between the cloud services and customers. TLS provides strong authentication, message privacy, and integrity (enabling detection of message tampering, interception, and forgery), interoperability, and algorithm flexibility.
+
+In addition to TLS, when you interact with Microsoft Energy Data Services, all transactions take place over HTTPS. 
+
+## Set up Customer Managed Keys (CMK) for Microsoft Energy Data Services Preview instance
+
+### Pre-requisites
+
+**Step 1- Configure the key vault**
+
+1.	You can use a new or existing key vault to store customer-managed keys. To learn more about Azure Key Vault, see [Azure Key Vault Overview](../key-vault/general/overview.md) and [What is Azure Key Vault](../key-vault/general/basic-concepts.md)?.
```

Comment:
As per the guideline, if the steps need to be executed in a particular sequence, use numbered list. Otherwise, it is suggested to use the bulleted list.